processor/datastore_test.go

```diff
@@ -74,7 +75,12 @@ func TestUpsert(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.Name, func(t *testing.T) {
-			store, err := newDatastore(":memory:")
+			db, err := sql.Open("sqlite3", ":memory:")
+			if err != nil {
+				t.Fatal(err)
+			}
```

Comment:
Maybe make a helper function for this so we don't have to write it three times?

Reply:
this is not needed as we would just end up wrapping sql.Open anyway